app/views/dishes_templates/index.html.erb

```diff
@@ -100,9 +100,10 @@
 
                 <td>
 
-                  <% if dishes.specialists[0] != nil%>
+                  <% if current_user.bookmarks.where("dish_id ="+dishes.id.to_s)[0] != nil%>
```

Comment:
Makes sense! Probably a way to do this a bit more cleanly, but I think this will work well.